Description: Changes the dependency management to use the system jars when building
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -1306,7 +1306,7 @@
 
     dependencies {
         compile project(":base"), BUILD_SRC
-        compile name: SWT_FILE_NAME
+        compile files("/usr/share/java/swt4.jar")
         stubCompile group: "junit", name: "junit", version: "4.8.2",
         project(":base").sourceSets.test.output, sourceSets.main.output
         antlr3 group: "org.antlr", name: "antlr", version: "3.1.3"
@@ -1687,7 +1687,7 @@
     }
     dependencies {
         compile BUILD_SRC, project(":base"), project(":graphics")
-        compile name: SWT_FILE_NAME
+        compile files("/usr/share/java/swt4.jar")
     }
 }
 
@@ -2050,13 +2050,13 @@
     classes << {
         // Copy all of the download libraries to libs directory for the sake of the IDEs
         File libsDir = rootProject.file("build/libs");
-        File antLib = new File(libsDir, "ant-1.8.2.jar")
+        File antLib = new File(libsDir, "ant-debian.jar")
         libsDir.mkdirs();
         for (File f : configurations.compile.files) {
             copy {
                 into libsDir
                 from f.getParentFile()
-                include "**/ant-1.8.2.jar"
+                include "**/ant-debian.jar"
                 includeEmptyDirs = false
             }
         }
--- a/buildSrc/build.gradle
+++ b/buildSrc/build.gradle
@@ -82,6 +82,7 @@
 
 dependencies {
     compile     group: "org.antlr", name: "antlr",          version: "3.1.3"
+    compile     group: "org.antlr", name: "stringtemplate", version: "3.2"
     testCompile group: "junit",     name: "junit",          version: "4.8.2"
     antlr3      group: "org.antlr", name: "antlr-runtime",  version: "3.1.3"
     antlr3      group: "org.antlr", name: "stringtemplate", version: "3.2"
